message_definitions/v1.0/common.xml

```diff
@@ -1342,6 +1371,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="224" name="MAV_CMD_DO_CONDITION_JUMP">
+        <description>Jump to the desired command in the mission list if condition is met</description>
```

Comment:
I don't see how this fits in the mission. The required airspace clearance is described on a nav delay command, but on this command it's assumed to already be known?